GoogleForCreators/web-stories-wp
# Animations: Prevent malformed HTML output

## Summary

Prevents animation markup from being stripped, especially for author users.

## Relevant Technical Choices

* KSES: Add required type attribute for animations. Added a unit test.
* Animations: Updates `AnimationOutput` to avoid printing `element` on the frontend for animations, avoiding potentially malformed HTML.

## To-do

Since there's also AMP sanitization happening later, we could think about extending our existing e2e test for author users.

## User-facing changes

No more broken animations for author users.

## Testing Instructions

1. Log in as author
1. Create new story from template (e.g. Beauty template)
1. Preview story on frontend
1. Verify that story & animation appears correctly

---



Fixes #5763
